lib/cc/services/asana.rb

```diff
@@ -1,6 +1,7 @@
 class CC::Service::Asana < CC::Service
   class Config < CC::Service::Config
-    attribute :api_key, Axiom::Types::String, label: "API key"
+    attribute :personal_access_token, Axiom::Types::String, label: "Personal Access Token"
```

Comment:
Sure, good call. I'll open an app PR that uses this branch and adds the obfuscation there